### ISSUE TITLE
Health data notification bug [1.4.0-16369] [linux-x64]

### ISSUE THREAD

**Issue by dvorapa** [adobe/brackets#11470]
If I do a clear installation of Brackets pre-release 1.4 and run Brackets after installation is done, a blue Health data notification appears. If I close it, the part of the screen where the notification was previously is unclickable. I e.g. can't click to start writing or in a smaller window size I even can't choose most of Help menu commands.

**Comment by abose** [comment]
@dvorapa can you go to debug logs and see the messages there.
in chrome: goto http://localhos:9234/ when brackets have the problem.

**Comment by dvorapa** [comment]
I haven't found any problem in a console. But I've found out that .healthdata-firstlaunch-container only changes its opacity to 0 (zero) and not its visibility or display. Below are screenshots of Dev Tools and Brackets before closing and after closing.

Before closing:
![snimek obrazovky porizeny 2015-07-25 02 30 16](https://cloud.githubusercontent.com/assets/5113916/8887074/8fd10452-3275-11e5-8552-c388390a62ce.png)
![snimek obrazovky porizeny 2015-07-25 02 30 33](https://cloud.githubusercontent.com/assets/5113916/8887078/9aae4060-3275-11e5-899b-d8d404b32429.png)
![snimek obrazovky porizeny 2015-07-25 02 30 41](https://cloud.githubusercontent.com/assets/5113916/8887079/9ee65a1e-3275-11e5-910e-f354f2b17132.png)

After closing:
![snimek obrazovky porizeny 2015-07-25 02 31 03](https://cloud.githubusercontent.com/assets/5113916/8887081/a545d128-3275-11e5-8a36-6fb5cc998ed2.png)
![snimek obrazovky porizeny 2015-07-25 02 31 09](https://cloud.githubusercontent.com/assets/5113916/8887086/ae0050fe-3275-11e5-9ca3-a8eeadfb2531.png)
![snimek obrazovky porizeny 2015-07-25 02 31 14](https://cloud.githubusercontent.com/assets/5113916/8887088/b1e1dc42-3275-11e5-8b5e-8eb7402b55ef.png)

**Comment by abose** [comment]
@prafulVaishnav .
There seems to be an error in loading the extension dir. That would cause an error in brackets starting up properly. Checkout the folder .config/Brackets/extensions/user

**Comment by dvorapa** [comment]
This error was caused by not reinstalling Brackets, just deleting the Brackets dir (including the extension dir) in order to achieve the first launch popup without reinstalling whole Brackets. But Brackets creates it at the startup. I'll fully reinstall Brackets to see, if there is the same error.

**Comment by dvorapa** [comment]
Yes, the error is thrown everytime I reinstall and first launch Brackets. But I don't understand, what have this error in common with behavior of not hiding the popup and making it just transparent (a new user then click into the area, where the popup was previously, and suddenly Brackets open url https://github.com/adobe/brackets/wiki/Health-Data).

**Comment by dvorapa** [comment]
v1.5 and v1.6 still broken
